pkg/dataplane/controller.go

```diff
@@ -399,6 +540,7 @@ func (c *Controller) syncNFTablesRules(ctx context.Context) error {
 				&expr.Cmp{Op: expr.CmpOpEq, Register: 0x1, Data: []byte{unix.IPPROTO_UDP}},
 				&expr.Payload{DestRegister: 0x1, Base: expr.PayloadBaseTransportHeader, Offset: 2, Len: 2},
 				&expr.Cmp{Op: expr.CmpOpEq, Register: 0x1, Data: binaryutil.BigEndian.PutUint16(53)},
+				&expr.Counter{},
```

Comment:
not directly related; if you're planning to keep this, split it out into a separate commit?

(but also, note that the reason that counters aren't automatic for nftables rules like they were with iptables is because there are performance implications (eg, cross-cpu cache invalidation or something).)